apps/sms/views/shared/js/navigation.js

```diff
@@ -751,6 +751,13 @@ var Navigation = {
       return Promise.resolve();
     }
 
+    if (navigationTransition) {
+      debug('startNavigation: previous navigation exists, deferring.');
```

Comment:
The reason why I did transition checking here before startNavogation is because startNavogation didn't check the currentPanel, and even we could check it inside startNavogation, it's harder to exist the navigation promise chain directly if we're in startNavogation already. So I add another checking in the toPanel.

BTW I forgot to rename the debug log... Will fix that later.